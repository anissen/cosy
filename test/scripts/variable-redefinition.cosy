fn bad() {
  let a = 'first'
  let a = 'second' // redefinition: should throw an error
  print a
}
bad()
