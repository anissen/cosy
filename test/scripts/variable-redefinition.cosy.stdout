â–  test/scripts/variable-redefinition.cosy, line 3:
1 | fn bad() {
2 |   let a = 'first'
3 |   let a = 'second' // redefinition: should throw an error
          ^ Variable with this name already declared in this scope.
4 |   print a
5 | }

